cmake/SetupPyInstall.cmake.in

```diff
@@ -2,8 +2,8 @@
 # Runs setup.py install
 #
 set(PYTHON_EXECUTABLE @Python3_EXECUTABLE@)
-set(core_version @core_version@)
-set(ENV{CYCLUS_CORE_VERSION} "${core_version}")
+set(CYCLUS_CORE_VERSION @CMAKE_PROJECT_VERSION@)
```

Comment:
If a user specifies `core_version` during setup, shouldn't we use that still?

Reply:
and don't we now set `CYCLUS_CORE_VERSION` reliably at the top level?

Reply:
Refactored build to differentiate between `CMAKE_PROJECT_VERSION`(`CYCLUS_PROJECT_VERSION` in `setup.py`) and `DATA_MODEL_VERSION`